packages/cdktf/lib/backends/backend_cos.ts

```diff
@@ -0,0 +1,24 @@
+import { Construct } from "constructs";
+import { TerraformBackend } from '../terraform-backend';
+import { keysToSnakeCase } from "../util";
+
+export class CosBackend extends TerraformBackend {
```

Comment:
Is there a particular reason the file is named `backend_cos` and the class `CosBackend`?

Reply:
I think it was from before I put them in their own folder. I'll rename.